components/Testimonial.vue

```diff
@@ -1,21 +1,118 @@
 <template>
```

Comment:
We should rename this component since it's not a singular testimonial.
Something like `testimonialsCarousel`,`testimonials`, `twitterTestimonialsCarousel`...

Reply:
haha, yeah at very least a plural, ne.